ViewStore.md

```diff
@@ -0,0 +1,135 @@
+# View Store
+
+View Store is an architecture pattern and a protocol used in SwiftUI development inspired by [The Composable Architecture](https://github.com/pointfreeco/swift-composable-architecture). A view store is an `ObservableObject` that allows us to separate view-specific logic and the rendering of a corresponding view in a way that is repeatable, prescriptive, flexible, and testable by default. It is available as a [Swift Package](https://github.com/Lickability/ViewStore).
+
+## The Protocol
+
+The protocol declaration itself is quite simple:
+
+```swift
+protocol ViewStore: ObservableObject {
+    associatedtype ViewState
+    associatedtype Action
+
+    var viewState: ViewState { get }
+    func send(_ action: Action)
+}
+```
+
+### `ViewState`
+
+The `viewState` property on the `ViewStore` protocol is the single source of truth for data that the corresponding `View` uses. It should always be declared as a `@Published` property. Similar to a [view model](https://github.com/Lickability/swift-best-practices/blob/main/ViewModel.md), the properties on `ViewState` should not require the corresponding `View` to perform any additional transformation logic or formatting.
```

Comment:
Formatting here is referring to things like text formatting (say, turning something like `let photosCount: Int` into `"Number of photos: \(photosCount)"`, and also date formatting, and number formatting like you mentioned. Basically, taking raw data and then formatting it for display, similar to our view models. Should I specifically call out something like "formatting for display (text, numbers, dates, etc.)"?

Reply:
It might be nice because my gut reaction was well I "format" how the view looks e.g. text styling etc. Might just be how my brain works so a simple e.g. text, number, date formatting might be good.

Reply:
Yeah, that’s fair. Will update.